tutorials/ComplexArithmetic/Workbook_ComplexArithmetic.ipynb

```diff
@@ -0,0 +1,953 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Success!\n"
+     ]
+    }
+   ],
+   "source": [
+    "# Run this cell using Ctrl+Enter (⌘+Enter on Mac).\n",
+    "from testing import exercise\n",
+    "from typing import Tuple\n",
+    "\n",
+    "import math\n",
+    "\n",
+    "Complex = Tuple[float, float]\n",
+    "Polar = Tuple[float, float]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Algebraic Perspective\n",
+    "\n",
+    "## Imaginary numbers\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### <span style=\"color:blue\">Exercise 1</span>: Powers of $i$.\n",
+    "\n",
+    "**Input:** An even integer $n$.\n",
+    "\n",
+    "**Goal:** Return the $n$th power of $i$, or $i^n$.\n",
+    "\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Solution \n",
+    "\n",
+    "When raising $i$ to a higher power then 2, the answer will not always be $-1$. For example, when raising $i$ to the power of 4 gives: $i^4 = i^2 \\cdot i^2$ which would give: $ -1 \\cdot -1 = 1$. \n",
+    "\n",
+    "A generalized rule could be made: when the power to which you raise $i$ is fully dividable by 4 the result will always be 1. This is the complete pattern that arises when taking $i$ to higher powers:\n",
+    "\n",
+    "|Power of I|$i^1$|$i^2$|$i^3$|$i^4$|$i^5$|$i^6$|$i^7$|$i^8$|$\\dots$|\n",
+    "|----|----|----|----|----|----|----|----|----|----|\n",
+    "|Result|$i$|$-1$|$i$|$1$|$i$|$-1$|$i$|$1$|$\\dots$|\n",
+    "\n",
+    "This means that $i^{(n \\cdot 4)}$ will always give $1$ if $n$ is a whole number.\n",
+    "\n",
+    "*Python note*:  \n",
+    ">`%` is the python modulo operator it returns the remainder of a division. For example: `7%2` gives 1, because 1 remains after $\\frac{7}{3}$, another example would be `10%5` which returns 0.  \n",
+    ">We can use this to know if $n$ in $i^n$, could be wholly divided by 4 for which we will use in the exercise.\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 2,
+   "metadata": {},
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "Success!\n"
+     ]
+    }
+   ],
+   "source": [
+    "@exercise\n",
+    "def imaginary_power(n : int) -> int:\n",
+    "    # If n is divisible by 4\n",
+    "    if n % 4 == 0:\n",
+    "        return 1\n",
+    "    else:\n",
+    "        return -1"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "[Return to task 1 of the Complex Arithmetic kata.](./ComplexArithmetic.ipynb#Exercise-1:-Powers-of-$i$.)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Complex Numbers\n",
```

Comment:
We probably don't need this section name in the workbook, we can just keep going through the exercises; the same goes for later section names "Geometric Perspective" and "Imaginary Exponents" (especially since the tasks of these two are sort of mixed together - tasks on conversion between Cartesian and polar forms, which look like geometric part, go between the exponents tasks)